elfpy/utils/apeworx_integrations.py

```diff
@@ -50,29 +52,28 @@ def get_transfer_single_event(tx_receipt: Receipt) -> ContractLog:
         ) from exc
 
 
-def get_pool_state(tx_receipt: Receipt, hyperdrive_contract: ContractInstance):
+def get_pool_state(tx_receipt: ReceiptAPI, hyperdrive_contract: ContractInstance):
     """
-    Aftering opening or closing a position, we query the smart contract for its updated pool info.
-    We return everything returned by getPoolInfo in the smart contracts, along with:
-        token_id: the id of the TransferSingle event (that isn't mint or burn), returned by get_transfer_single_event
-        block_number_: the block number of the transaction
-        prefix_: the prefix of the trade (LP, long, or short)
-        maturity_timestamp: the maturity time of the trade
+    Return everything returned by `getPoolInfo()` in the smart contracts, along with:
```

Comment:
This is a confusing sentence. Along with what? Are you opposed to also adding "After opening or closing a position, we query the smart contract for its updated pool info." to the brief description?

Reply:
the full docstring reads:
```
    Return everything returned by `getPoolInfo()` in the smart contracts, along with:

    Additional information includes:

    * `token_id` : the id of the TransferSingle event (that isn't mint or burn), returned by `get_transfer_single_event`
    * `block_number_` : the block number of the transaction
    * `prefix_` : the prefix of the trade (LP, long, or short)
    * `maturity_timestamp` : the maturity time of the trade
  ```

Reply:
agree the `along with:` shouldn't be there, removing it